src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -366,4 +366,25 @@
   <data name="net_http_feature_UWPClientCertSupportRequiresCertInPersonalCertificateStore" xml:space="preserve">
     <value>Client certificate was not found in the personal (\"MY\") certificate store. In UWP, client certificates are only supported if they have been added to that certificate store.</value>
   </data>
+  <data name="net_http_content_unexpected_end" xml:space="preserve">
+    <value>Encountered an unexpected end of the content stream.</value>
+  </data>
+  <data name="net_http_headers_missing_location" xml:space="preserve">
+    <value>The response code indicates a redirect, but the 'Location' header is missing.</value>
+  </data>
+  <data name="net_http_invalid_chunk_size" xml:space="preserve">
+    <value>The response stream contains an invalid chunk size.</value>
```

Comment:
For now, I think we don't need this value, and can just use https://github.com/steji113/corefx/blob/3c54c1b9f3da8c5022eb5256a98e977f9de4c7d2/src/System.Net.Http/src/Resources/Strings.resx#L331.  If we want to subsequently go through and create specific reasons for each of errors we detect, we could do that (as long as it doesn't negatively, measurably impact perf), but I think it's a bit odd we're currently special-casing just this one response error.